ui/app/src/elements/dashboard/we-group-home.ts

```diff
@@ -276,46 +276,43 @@ export class WeGroupHome extends ScopedElementsMixin(LitElement) {
         </div>
         `,
       complete: (info) => { return html`
-          <nh-profile-prompt>
-            <div slot="hero">
-              <div>
-                <div class="column center-content">
-                  <img
-                    class="we-logo"
-                    style="margin-top: 30px;"
-                    src=${info.logoSrc!}
-                  />
-                  <div
-                    style="font-weight: bold; margin-top: 20px; font-size: 1.2em;"
-                  >
-                    ${info.name}
-                  </div>
-                  <div
-                    style="margin: calc(1px * var(--nh-spacing-md)); margin-top: calc(1px * var(--nh-spacing-sm)); font-size: calc(1px * var(--nh-font-size-lg))"
-                  >
-                    How would you like to appear in this neighbourhood?
+            <with-profile .component=${"prompt"} .weGroupId=${this.weGroupId} style="display: flex; flex: 1;">
+                <div slot="hero" style="color: var(--nh-theme-fg-default)">
+                  <div class="column center-content">
+                    <img
+                      class="we-logo"
+                      style="margin-top: 30px;"
+                      src=${info.logoSrc!}
+                    />
+                    <div
+                      style="font-weight: bold; margin-top: 20px; font-size: 1.2em;"
+                    >
+                      ${info.name}
+                    </div>
+                    <div
+                      style="margin: calc(1px * var(--nh-spacing-md)); margin-top: calc(1px * var(--nh-spacing-sm)); font-size: calc(1px * var(--nh-font-size-lg))"
+                    >
+                      How would you like to appear in this neighbourhood?
+                    </div>
                   </div>
-                </div>
-              </div>
-            </div>
-
-            <div slot="info">
-              ${this.renderJoinErrorSnackbar()} ${this.renderInstallingProgress()}
-              ${this.renderSuccessSnackbar()}
-            </div>
-            <div slot="content" class="profile-prompt-content">
-              ${this.renderContent()}
-            </div>
-            
-            </nh-profile-prompt>
+                            </div>
+                
+                            <div slot="info">
+                ${this.renderJoinErrorSnackbar()} ${this.renderInstallingProgress()}
+                ${this.renderSuccessSnackbar()}
+                            </div>
+                            <div slot="content" style="display: flex; flex: 1;">
+                ${this.renderContent()}
+                            </div>
+            </with-profile>
         `}
     })
   }
 
   static get scopedElements() {
```

Comment:
@pospi see above. This would have been one that has not been switched out. Didn't think about that with static declarations though, will aim to keep this consistent.